### PR TITLE
Complete task 6

### PR DESCRIPTION
**Pull request by JackAshwell11**
Tasks:
- [x] Complete documentation.
- [x] Add BlueJ files.

Don't merge this pull request until Task 5 is done so I can merge that into here and finish the documentation.

### CODE REVIEW

**Comment by 1Klevi1** [comment]
ye it's alright

File: doc/search-page.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
```
Comment (1Klevi1) [inline doc/search-page.js:4]: why u didn't delete the newlines here?

File: doc/script.js [hunk 2]
```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2013, 2022, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
```
Comment (1Klevi1) [inline doc/script.js:6]: also here

**Review by 1Klevi1 — APPROVED** [review]
some files had some new lines, I did comment two of them, but everything was cool, by the way how did u create those doc file with javasript, html and css. Was there  plugin or smth like that, cuz it was noice

File: doc/search-page.js (hunk 1 above)
Comment (JackAshwell11) [reply doc/search-page.js:4]: They're autogenerated by Javadoc so its not really our problem

File: doc/script.js (hunk 2 above)
Comment (JackAshwell11) [reply doc/script.js:6]: They're autogenerated by Javadoc so its not really our problem

**Pull request merged**